test/contract based tests/LosslessController.js

```diff
@@ -10,7 +11,7 @@ let env;
 
 const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
```

Comment:
Maybe we can move this to some test utils file that would contain all of such helper constants and functions.

Reply:
Addressed on 0137e8c